src/Translation/Translator.php

```diff
@@ -0,0 +1,138 @@
+<?php
```

Comment:
since this PR is targeted for 4.5.0 (same as #1077) and is already using php 7 features like the `string` parameter type hints it should fully make use of it. 

right here `declare strict` declaration is missing